runtime/jcl/common/mgmtthread.c

```diff
@@ -1287,8 +1291,28 @@ freeThreadInfos(J9VMThread *currentThread, ThreadInfo *allinfo, UDATA allinfolen
 		while (sinfo) {
 			next = sinfo->next;
 			j9mem_free_memory(sinfo);
+			if (sinfo->obj.unsafe != NULL) {
+				vmfns->j9jni_deleteLocalRef((JNIEnv *)currentThread, (jobject) sinfo->obj.unsafe);
+			}
```

Comment:
Constants (i.e. `NULL`) on left of `!=`.

Reply:
done. btw, looks like there are both types of usages in this file, for example: https://github.com/eclipse-openj9/openj9/blob/674096f441933f2a7cef9c172612a874e85ab6bd/runtime/jcl/common/mgmtthread.c#L1725 , and that can be another PR!

Reply:
I'm fine either way; you could fix up those other places in this or another pull request.